pravega/src/main/resources/reference.conf

```
@@ -0,0 +1,79 @@
+akka.alpakka.pravega {
+  #
+  # ClientConfig (Pravega internals)
+  defaults.client-config {
+    # ControllerURI The controller rpc URI. This can be of 2 types
+    #  1. tcp://ip1:port1,ip2:port2,...
+    #   This is used if the controller endpoints are static and can be directly accessed.
+    #  2. pravega://ip1:port1,ip2:port2,...
+    #   This is used to autodiscovery the controller endpoints from an initial controller list.
+    #controller-uri = "tcp://localhost:9090"
+
+    # An optional property representing whether to enable TLS for client's communication with the Controller.
+    # If this property and enable-tls-to-segment-store are not set, and the scheme used in controller-uri
+    #  is "tls" or "pravegas", TLS is automatically enabled for both client-to-Controller and
+    # client-to-Segment Store communications.
+    #enable-tls-to-controller = false
+
+    # An optional property representing whether to enable TLS for client's communication with the Controller.
+    # If this property and 'enable-tls-to-controller' are not set, and the scheme used in 'controller-uri'
+    #   is "tls" or "pravegas", TLS is automatically enabled for both client-to-Controller and
+    # client-to-Segment Store communications.
+    #enable-tls-to-segment-store = false
+
+    # Maximum number of connections per Segment store to be used by connection pooling.
+    #max-connections-per-segment-store=10
+
+    # Path to an optional truststore. If this is null or empty, the default JVM trust store is used.
+    # This is currently expected to be a signing certificate for the certification authority.
+    #trust-store
+
+    # If the flag 'isEnableTls'  is set, this flag decides whether to enable host name validation or not.
+    #validate-host-name=true
+  }
+  reader {
+    client-config = ${akka.alpakka.pravega.defaults.client-config}
+    # ReaderConfig (Pravega internals)
+    config {
+      #disable-time-windows = false
+      #initial-allocation-delay = 0
+    }
+    timeout = 5000
```

```suggestion
    timeout = 5 seconds
```

(once settings load a duration instead of a long)